lstchain/data/data_selection_cuts.json

```diff
@@ -0,0 +1,15 @@
+{
+  "general":{
+    "gh_score": 0.8,
+    "source_fov_offset": 2.83,
+    "intensity": 200,
+    "r": 1,
+    "wl": 0.1,
+    "leakage2_intensity": 0.2,
```

Comment:
does this parameter still exist?

Reply:
To use it for pre v0.6 format files. I have created a condition in the data_clean.py script to account for it.

Reply:
No, it is the old name of the parameter one line below